src/veupath/redmine/client/checks/check_genome_issues.py

```diff
@@ -254,6 +257,7 @@ def main():
     parser.add_argument("--store", type=str, help="Write json files for each Redmine issue")
 
     # Optional
+    parser.add_argument("--ensembl_version", type=int, help="Provide Ensembl API version (for the report")
```

Comment:
@MatBarba I know I'm too late, but there is a typo in the `help` text (missing `)`).

Reply:
I'll add it, good catch